data/src/main/java/com/linkedin/data/schema/SchemaToPdlEncoder.java

```diff
@@ -756,7 +756,7 @@ private void writeDocAndProperties(RecordDataSchema.Field field) throws IOExcept
    * @param inlinedTypes cumulative set of all inlined types in this schema (and its descendents).
```

Comment:
We should clarify that this set is a collection of simple names, not full names.